src/NServiceBus.AcceptanceTesting/Support/TaskExtensions.cs

```diff
@@ -8,6 +8,7 @@
 
     static class TaskExtensions
     {
+        //this method will not timeout a task if the debugger is attached.
```

Comment:
For me the comment is enough. I see @dbelcham arguments and I think it is overkill